src/documentation/pages/components-wip/forms/search-input/_properties.vue

```diff
@@ -31,6 +31,30 @@
               td
                 div
                   code false
+            tr
```

Comment:
Don't need to add anything here as this is for the Vue component not for the web component which is automatically updated.